### ISSUE TITLE
There is no name property for all the Slider in the ScrollViewer(Preview) pane.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#185]
**Describe the bug**

**Pre-requisite: **
Turn on Narrator.
Set narrator to item mode.

**Steps to reproduce the bug**

1. Launch XAML Control Gallery app.
2. Navigate to Scroll Viewer button and activate it.
3. Navigate to any slider e.g ZoomFactor and observe the Narrator reading.

**Expected behavior**

All Sliders should have a name property.

**Actual behavior**

When user navigate to any slider, narrator reads only percentage value. e.g "Slider 18%".

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63895947-859daa00-c9a5-11e9-8d8f-4e54136a7f0a.png)

![image](https://user-images.githubusercontent.com/13142665/63895963-8fbfa880-c9a5-11e9-9b4c-0996c9dfd045.png)

**Version Info**

Gallery version (found on Settings page): 

Version Info: 1.2.8.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Coped from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/22205775/?triage=true)
